src/components/select/Select.tsx

```diff
@@ -32,10 +103,17 @@ const DropdownIndicator = ({ selectProps: { inverse } }) => {
     </Icon>
   );
 };
-
-const ClearIndicator = ({ innerProps, selectProps: { inverse } }) => {
+//
+const ClearIndicator = <IsMulti extends boolean>(clearIndicatorProps: ClearIndicatorProps<OptionType, IsMulti>) => {
+  // @ts-ignore
+  const inverse = clearIndicatorProps.selectProps.inverse;
```

Comment:
Does this actually work? I don't see this `inverse` being passed to the react-select component anywhere 🤔